src/agent/samplers/cpu/linux/bandwidth/stats.rs

```diff
@@ -10,11 +10,11 @@ use crate::agent::*;
 pub static CGROUP_CPU_BANDWIDTH_QUOTA: GaugeGroup = GaugeGroup::new(MAX_CGROUPS);
 
 #[metric(
-    name = "cgroup_cpu_bandwidth_period",
+    name = "cgroup_cpu_bandwidth_period_duration",
```

Comment:
What was the reason for renaming this? That's a pretty verbose metric name now.

Reply:
Becasue it's very similar with the name of the new metric `cgroup_cpou_bandwidth_periods and the cgroup_cpu_bandwidth_period shows the duration of a period.